Lib/test/libregrtest/main.py

```diff
@@ -21,6 +21,55 @@
 from test import support
 
 
+# Set of tests run by default if --pgo is specified.  The tests below were
+# chosen based on the following criteria: either they exercise a commonly used
+# C extension module or type, or they run some relatively typical Python code.
+# Long running tests should be avoided because the PGO instrumented executable
+# runs slowly.
```

Comment:
I would prefer to keep main.py small. Would you mind to create libregrtest/pgo.py for this list please?

Reply:
Ok.